doc/installation.md

```diff
@@ -88,6 +88,7 @@ following command:
 
 ```
 docker run -it -w /kikit -v $(pwd):/kikit yaqwsx/kikit /bin/bash
+docker run -it -w /kikit -v %cd%:/kikit yaqwsx/kikit /bin/bash
```

Comment:
Please, add a note on which line belongs to Linux and which one belongs to Windows. I think it would be best to have in separate verbatim environemnts.